saleor/order/models.py

```diff
@@ -327,6 +327,14 @@ def get_total(self):
     def quantity_unfulfilled(self):
         return self.quantity - self.quantity_fulfilled
 
+    @property
+    def is_digital(self) -> bool:
```

Comment:
Not sure if we should combine two checks (`is_digital` and `is_empty`) into one, but if it's reasonable, then ok.

Reply:
What do you propose? 
OrderLine is digital only when a variant is digital and variant has already attached a file.

Reply:
As I said, if there is a valid reason, it may stay as it is. Looks like there is, so no changes I guess!